### ISSUE TITLE
Suggestion: Capture Approved/Accepted Trades?

### ISSUE THREAD

**Issue by conetopia** [Kyusung4698/PoE-Overlay#743]
Version 1.0.2 +
In the long run, A video capture could be a thing.
However, What I'm looking for is.... Every time someone messages me for a trade...
You know how it shows up on the top left corner of screen?
It would be really cool if you could manually click on that trade, or hit a hotkey ect.
That would basically Take a picture of the stats of that item, and put it in your pictures folder.

Kinda like the way it shows up in the Market window, basically a simple list.
That way I can kinda keep crafting things, and see if there was anything in common that people seem to be interested in.

**Comment by Kyusung4698** [comment]
Hi, thanks for your suggestion. For now i'm only getting the item name in the trade message. So maybe a extra hotkey which you can press while hovering the item before you trade it to copy the stats and add those to some kind of data pool to generate different statistics later. Seems like a neat idea.

BTW most windows can be dragged by the menu bar at the top. I personally placed it at the left corner of the minimap.

**Comment by conetopia** [comment]
> 
> 
> Hi, thanks for your suggestion. For now i'm only getting the item name in the trade message. So maybe a extra hotkey which you can press while hovering the item before you trade it to copy the stats and add those to some kind of data pool to generate different statistics later. Seems like a neat idea.
> 
> BTW most windows can be dragged by the menu bar at the top. I personally placed it at the left corner of the minimap.

Yeah i noticed windows can be dragged, Nice to see that they save in place, so they are still in position Upon re-launch ect.


But yeah I'm not sure if your familiar with Diablo 3's Item collage back in the day.
But basically POE Acquisition had a similar feature, where it reads all the info of the item...
Then you can click a button so it basically takes a picture and Uploads it automatically to imgur.
However in this case, could just do picture folder.

But yeah basically what ever item your hovering over or about to trade ect...
you could maybe make a button for Quick-Snap or something.
Result idea would look like the picture.

![te](https://user-images.githubusercontent.com/4310295/84875631-d7eeab00-b04b-11ea-9820-342f466c077e.png)
